core/include/esp/registers.h

```diff
@@ -14,295 +14,33 @@
 #ifndef _ESP_REGISTERS
 #define _ESP_REGISTERS
 #include "common_macros.h"
+#include "esp/types.h"
 
-typedef volatile uint32_t *esp_reg_t;
-
-/* Internal macro, only defined in header body */
-#define _REG(BASE, OFFSET) (*(esp_reg_t)((BASE)+(OFFSET)))
+#include "esp/iomux_regs.h"
+#include "esp/gpio_regs.h"
+#include "esp/timer_regs.h"
+#include "esp/wdt_regs.h"
+#include "esp/dport_regs.h"
 
 /* Register base addresses
 
    You shouldn't need to use these directly.
  */
 #define MMIO_BASE   0x60000000
-#define DPORT_BASE  0x3ff00000
+//#define DPORT_BASE  0x3ff00000
```

Comment:
Yup.. the *_BASE defines for those are now included in the corresponding *_regs.h file.  I considered just deleting them, but I think commenting them out makes for a handy "checklist" until we finish with all of them, so decided to just leave them commented..